src/Interpreters/executeDDLQueryOnCluster.cpp

```diff
@@ -316,6 +318,8 @@ DDLQueryStatusSource::DDLQueryStatusSource(
     {
```

Comment:
Shouldn't we also add `|| DistributedDDLOutputMode:: THROW_ONLY_ACTIVE` to `throw_on_timeout`?